codalab/worker/bundle_service_client.py

```diff
@@ -179,5 +179,6 @@ def get_bundle_contents(self, uuid, path):
             '/bundles/' + uuid + '/contents/blob/' + path,
             headers={'Accept-Encoding': 'gzip'},
             return_response=True,
+            timeout_seconds=10 * 60,
```

Comment:
Should this be set to `URLOPEN_TIMEOUT_SECONDS`?

Reply:
I think the intention was to make the timeout a bit more lenient for `/bundles/conents/blob/`

Reply:
Does this timeout mean that the download has to take less than `10*60` seconds?

Reply:
Perhaps we set it to `URLOPEN_TIMEOUT_SECONDS * 2` instead then?

Reply:
Updated